src/ucp/api/ucp.h

```diff
@@ -1620,6 +1620,25 @@ ucs_status_t ucp_ep_rkey_unpack(ucp_ep_h ep, void *rkey_buffer, ucp_rkey_h *rkey
 
 /**
  * @ingroup UCP_MEM
+ * @brief Get a local pointer to remote memory.
+ *
+ * This routine returns a local pointer to the remote memory described
+ * by the rkey.
```

Comment:
please add a @note that the function will not work for all endpoints but only for a class of endpoints that are reachable through shared memory